src/components/ContestTile.js

```diff
@@ -59,23 +58,25 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
           </p>
         )}
         <ClientOnly>
-          <Link
-            to={contestPath}
+          <a
+            to={fields?.contestPath || "/"}
```

Comment:
When would contestPath be undefined? Also,

```suggestion
            href={fields?.contestPath || "/"}
```